Makefile

```diff
@@ -70,5 +70,9 @@ update-file-perms:
 	hack/make-rules/update/file-perms.sh
 verify-file-perms:
 	hack/make-rules/verify/file-perms.sh
+update-spelling:
+	hack/make-rules/update/misspell.sh
+verify-spelling:
+	hack/make-rules/verify/misspell.sh
```

Comment:
these should be under `.PHONY` (same comment as the other PR https://github.com/kubernetes/test-infra/pull/24213#discussion_r741517101)

Reply:
done